rtrlib/rtr/packets.c

```diff
@@ -947,6 +981,38 @@ static int rtr_store_router_key_pdu(struct rtr_socket *rtr_socket, const void *p
 	return RTR_SUCCESS;
 }
 
+/*
+ * @brief Appends the ASPA pdu to ary.
+ *
+ * @return RTR_SUCCESS On success
+ * @return RTR_ERROR On realloc failure
+ * @attention ary is not freed in this case, because it might contain data that is still needed
+ */
+static int rtr_store_aspa_pdu(struct rtr_socket *rtr_socket, const void *pdu, const unsigned int pdu_size,
+					struct pdu_aspa **ary, unsigned int *ind, unsigned int *size)
```

Comment:
suggestion: `pdu_aspa_array` instead of `ary`

Reply:
I'm going to refactor the entire function